### DISCUSSION TITLE
How to customize HTML file generated for React?

### DISCUSSION THREAD

**Discussion by runoncedev** [web-infra-dev/rspack#5115] [Q&A] [answered]
For example, if I want to add some attribute to the `<html>` element or add something inside the `<head>`.

Looked into `pluginReact()` options but there doesn't seem to be anything related to this.

My `rsbuild.config.ts`:

```
import { defineConfig } from "@rsbuild/core";
import { pluginReact } from "@rsbuild/plugin-react";

export default defineConfig({
  plugins: [pluginReact()],
});
```

Using:

```
"@rsbuild/core": "^0.2.9",
"@rsbuild/plugin-react": "^0.2.9",
```

**Comment by xc2** [comment, marked as answer]
please check out https://rsbuild.dev/config/html/template
